src/editors/containers/VideoEditor/components/VideoSettingsModal/components/ThumbnailWidget/index.test.jsx

```diff
@@ -52,22 +52,22 @@ describe('ThumbnailWidget', () => {
     });
     test('snapshots: renders as expected with a thumbnail provided', () => {
       expect(
-        shallow(<ThumbnailWidget {...props} thumbnail="sOMeUrl" videoType="edxVideo" />),
+        shallow(<ThumbnailWidget {...props} thumbnail="sOMeUrl" videoId="e0466f70-9fb0-40d0-bcd0-3a3f6e9fa890" />),
```

Comment:
nit: 
```suggestion
        shallow(<ThumbnailWidget {...props} thumbnail="sOMeUrl" videoId="sOmEID" />),
```
and then we mock out isEdxVideo